quality-eye/README.md

```diff
@@ -0,0 +1,5 @@
+# Quality Eye - NGE2E
+
+NGE2E is the next generation End-to-End Testing framework that aims to help developers to set up, debug, and verify E2E tests with ease. It’s built based on the lessons learnt from tens of hundreds of test cases in the SkyWalking main repo.
```

Comment:
From the validation part, https://skywalking.apache.org/blog/e2e-design/#verify, this is not a common usage e2e test tool.
Do you plan to expand the scope?

Reply:
[This](https://github.com/apache/skywalking-website/pull/184) should address your question